holmes/plugins/toolsets/mcp/toolset_mcp.py

```
@@ -135,3 +219,79 @@ async def _get_server_tools(self):
+        ):
+            async with ClientSession(read_stream, write_stream) as session:
+                _ = await session.initialize()
+                return await session.list_tools()
+
+    def get_example_config(self) -> Dict[str, Any]:
+        return {"command": "python", "args": ["/path/to/mcp_server.py"]}
+
+
+def get_mcp_toolset_from_config(config: dict[str, Any], name: str) -> BaseMCPToolset:
+    if not config:
+        raise ValueError("Config must not be empty")
```

Ruff (0.13.1)

Avoid specifying long messages outside the exception class

(TRY003)